BranchSDK/src/BranchIO/Branch.cpp

```diff
@@ -248,28 +246,48 @@ Branch::setIdentity(const String& userId, IRequestCallback* callback) {
 void
 Branch::logout(IRequestCallback *callback) {
     if (getSessionInfo().hasSessionId()) {
-        IdentityLogoutEvent event;
-        event.setResultHandler([this](const JSONObject& result) {
-            IStorage& storage(Storage::instance());
-            storage.remove("session.identity");
-            if (result.has(Defines::JSONKEY_SESSION_ID)) {
-                auto sessionId = result.getNamedString(Defines::JSONKEY_SESSION_ID);
-                getSessionInfo().setSessionId(sessionId);
+        if (getAdvertiserInfo().isTrackingDisabled()) {
+            if (callback) {
+                callback->onStatus(0, 0, "Requested operation cannot be completed since tracking is disabled");
+                callback->onError(0, 0, "Tracking is disabled");
             }
-            if (result.has(Defines::JSONKEY_SESSION_RANDOMIZED_BUNDLE_TOKEN)) {
-                auto bundleToken = result.getNamedString(Defines::JSONKEY_SESSION_RANDOMIZED_BUNDLE_TOKEN);
-                getSessionInfo().setBundleToken(bundleToken);
-                storage.setString("session.randomized_bundle_token", bundleToken);
-            }
-        });
-        sendEvent(event, callback);
+            return;
+        }
+        
+        IStorage& storage(Storage::instance());
+        storage.remove("session.identity");
```

Comment:
@echo-branch  Its removing from storage.

Reply:
Ok. I think I figured out why the behavior was different from expectations. On windows we don't support setting identity prior to install event?